internal/pkg/cli/svc_deploy.go

```diff
@@ -431,6 +449,7 @@ func buildSvcDeployCmd() *cobra.Command {
 	cmd.Flags().StringVar(&vars.imageTag, imageTagFlag, "", imageTagFlagDescription)
 	cmd.Flags().StringToStringVar(&vars.resourceTags, resourceTagsFlag, nil, resourceTagsFlagDescription)
 	cmd.Flags().BoolVar(&vars.forceNewUpdate, forceFlag, false, forceFlagDescription)
+	cmd.Flags().BoolVar(&vars.disableRollback, noRollbackFlag, false, noRollbackFlagDescription)
```

Comment:
awesome feature! Do we need to add this flag to `cli/deploy.go` as well?

Reply:
I am internally debating on whether to add the flag for job as well (and hence `cli/deploy.go`). I felt like this was a niche flag which we don't encourage of most of the time 🤔

Reply:
I agree that it's not common, but it feels like people might want to troubleshoot they job deployments or any other kind of deployment.

I'm down to merge this PR if we can do a follow-up for `job deploy` and `deploy` what do you think?